soc/arm/nxp_imx/rt6xx/soc.c

```diff
@@ -159,6 +159,14 @@ static ALWAYS_INLINE void clock_init(void)
 	CLOCK_AttachClk(kFFRO_to_FLEXCOMM5);
 #endif
 
+#if (DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(flexcomm1), nxp_lpc_i2s, okay)) && \
+	(DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(flexcomm3), nxp_lpc_i2s, okay))
+	/* attach AUDIO PLL clock to FLEXCOMM1 (I2S1) */
+	CLOCK_AttachClk(kAUDIO_PLL_to_FLEXCOMM1);
```

Comment:
Might be better to write this as:
```
#if ... flexcomm1...
  CLOCK_AttachClk(kAUDIO_PLL_to_FLEXCOMM1);
#endif
#if ... flexcomm2...
  CLOCK_AttachClk(kAUDIO_PLL_to_FLEXCOMM2);
#endif
...
```

Reply:
Sure, I will make this change

Reply:
@mmahadevan108  just reminder this needs changed.